extensions/sql-bindings/src/test/common/azureFunctionsUtils.test.ts

```diff
@@ -349,13 +343,230 @@ describe('AzureFunctionUtils', function (): void {
 
 			let result = await azureFunctionsUtils.promptForObjectName(BindingType.input, connectionInfo);
 
-			should(promptStub.notCalled).be.true('showInputBox should not have been called');
+			should(inputBoxStub.notCalled).be.true('showInputBox should not have been called');
 			should(quickPickStub.calledTwice).be.true('showQuickPick should have been called twice');
 			should(connectionInfo.database).be.equal('testDb', 'Should have connectionInfo.database to testDb after user selects database');
 			should(result).be.equal('[schema].[testTable]', 'Should return [schema].[testTable] since user selected table');
 		});
 	});
 
+	describe('PromptAndUpdateConnectionStringSetting', function (): void {
```

Comment:
I only see tests for : 

1. No local.settings.json file existing (fs.access rejects)
2. local.settings.json having non-known values in the settings

You should be adding tests with 

1. Empty values section
2. Values section that only has "known" values

Since that's a core part of the logic you're adding.